### DISCUSSION TITLE
RNAseq workflow

### DISCUSSION THREAD

**Discussion by chuddy-ibk** [biostars/biostar-handbook-issues#256] [Q&A] [answered]
Dear Community, 

I am trying to get through the RNAseq workflow, however I got stuck several times and now I think I should stop completely and ask for help before trying to skip too much. 

1)  when I try to carry out R scripts via "RScript" nothing works. I get that this command was found found but there is an option to download  "Rscript" from deb-base-core (4.1.2-1ubuntu2). For the pase_featurecount.r i just carried out the script via RStudio and it worked. Nevertheless, I wonder why the terminal approach won't do. 

2) When I try to combine_transcripts.r within RStudio, nothing work but an error is put out:

"
> # Load the packages
> library(tximport)
Error in library(tximport) : there is no package called ‘tximport’
"

I tried to somehow differently get  tximport but was not succhessfull as this error persisted. my working directory should be the same as back than with the first script I used succesfully in 1). 

3) In the next step I also tried differential expression, without succuess. Same error as in 2). 

"
> # Load the library.
> suppressPackageStartupMessages(library(DESeq2))
Error in library(DESeq2) : there is no package called ‘DESeq2’
"

So I guess I cannot go on from here without help. 
Any ideas? 

Best
Tom

**Comment by ialbert** [comment]
1. `Rscript` is part of R, when you install R you  are also installing Rscript as well. So if you can run R then Rscript should run as well. Can you run R separately? 

If you are using RStudio then you will first need to complete the package installation in Rstudio as well. There are instructions on what to run in RStudio before you run the scripts.

I would guess that in both cases the installation instructions were not quite performed as described in the book.

**Reply by chuddy-ibk** [reply]
Thank you for your reply. 

R was working perfectly. Anyway, I deinstalled everything (also Rstudio) and only use R in the bioinfo env, where it is included. 
My mistake was to copy-paste the "RScript" code, as "Rscript" would have been the solution - only using a small s instead of the capital one. 
Thus, it worked now. Did they change from the capital S to the small s over the last update of the book? 
Wishing you a great day!

**Reply by ialbert** [reply, marked as answer]
I apologize for the typo; the UNIX version that runs in MacOS is case insensitive; hence the commands also work when capitalized differently.

This makes capitalization typos more difficult to catch when working via MacOS, I will correct the error.

**Comment by njbowen** [comment]
this can probably be marked as answered, correct?

**Reply by ialbert** [reply]
yes, correct, I did so now
